package.json

```diff
@@ -11,6 +11,7 @@
     "uglify-js": "*",
     "autoprefixer": "*",
     "roole": "*",
-    "react-tools": "*"
+    "react-tools": "*",
+    "clean-css": "3.4.5"
```

Comment:
@kriswallsmith you're right, minor version should be updatable; I was worried about future breaking changes but I guess those will go in `3.5.*`; will submit a PR soon to correct this